code/game/atoms.dm

```diff
@@ -144,6 +144,9 @@
 /atom/proc/emp_act(severity)
 	return
 
+/atom/proc/microwave_act(obj/machinery/microwave/M)
+	return
+

```

Comment:
How come?

Also this should probably be a /obj/item/proc. Makes no sense for everything to be microwavable: it is kinda hard to microwave /turf